build/yaml/steps/linux/binaries.yaml

```diff
@@ -5,11 +5,13 @@
 
 steps:
 # Login to ACR
-- task: Docker@2
+- task: AzureCLI@2
   displayName: Docker Login
```

Comment:
>Docker [](http://example.com/codeflow?start=15&length=6)

nit rename

Reply:
I'm going to leave as-is since it is stilling providing credentials to the docker service.